### DISCUSSION TITLE
WIDE=1 oversaturation on HERO11

### DISCUSSION THREAD

**Discussion by RikshaDriver** [gopro/labs#636] [General]
Having done some testing between the HERO12 and HERO11 Black, I've observed that the HERO11 WIDE=1 is far more oversaturated when compared to the HERO12. This can be confirmed in the vectorscope with the uncorrected sources. Once corrected, all colors on HERO11 are extremely oversaturated.

Left is HERO12 and right is HERO11:

<img width="324" alt="hero12_vectorscope" src="https://github.com/gopro/labs/assets/5790757/f53f21d0-6135-4c5e-925d-23a245a47885"><img width="320" alt="hero11_vectorscope" src="https://github.com/gopro/labs/assets/5790757/b39160c5-d2cf-4ec3-9e70-8fe88821d2e2">

Sample clips below for reference:

HERO12 Clip:
![HERO12_WIDE](https://github.com/gopro/labs/assets/5790757/d39e0402-21a5-4270-9f55-ddf93e643431)

HERO11 Clip:
![HERO11_WIDE](https://github.com/gopro/labs/assets/5790757/ef0e013d-3d9a-4b36-a92a-d6d5ddff158c)

**Comment by dnewman-gpsw** [comment]
Was one camera in Natural/Vibrant and the other in Flat. Those modes are still active, even with LOGB and WIDE extensions running.  You can tell by looking at the metadata.

**Reply by RikshaDriver** [reply]
Definitely Protune Flat with LOGB400 on HERO11 and GP-Log on HERO12 - I had double checked metadata before raising this.

HERO11:
>   PRTN Y
  PTWB AUTO
  PTSH LOW
  **PTCL FLAT**
  EXPT AUTO
  PIMX 6400
  PIMN 100
  PTEV 0.0
  STRM 
    EMPT 2
    BASE H111_[yyyymmdd]_
    LOGB 400
    EXPS 30
    EXPX 48
    NR01 1
    DLRV 1
    24HZ 1
    WIDE 1
    BITR 150
    EVBS 1

HERO 12:

>   PRTN Y
  PTWB AUTO
  PTSH LOW
  PTCL LOG
  EXPT AUTO
  PIMX 400
  PIMN 100
  PTEV 0.0
  STRM 
    EMPT 4
    BASE H121_[yyyymmdd]_
    WIDE 1
    EXPS 1
    NR01 1
    EXPX 48
    PRXY 4
    BITR 150
    24HZ 1
    EVBS 1

**Reply by dnewman-gpsw** [reply]
Put them both in Flat with LOGB=400, the GP-Log might have addition tweaks that I'm not aware of.   WIDE only disables the color saturation matrix, not any other processing stages.

**Reply by RikshaDriver** [reply]
Did a quick test on the H12 and there are definitely some differences between GP-Log and LOGB=400.

It appears GP-Log is desaturating the colors ever so slightly whereas LOGB=400 is oversaturating.

Interesting.
